ci.sh

```diff
@@ -1,21 +1,30 @@
 #!/bin/bash
 set -e
+# when running in github actions:
 RVM_PATH=/home/runner/.rvm
-ARGS=("$@")
+# when running locally using act:
+#RVM_PATH=/usr/local/rvm
+MATRIX_RUBY_VERSION=$1
+MATRIX_RAILS_VERSION=$2
 
 if [[ ! -d $RVM_PATH ]]; then
-  curl -sSL https://get.rvm.io | bash
+  # this fetches the develop version; using -s stable should fetch the latest stable, but has a gpg error:
+  curl -sSL https://get.rvm.io | bash -s
 fi
 
+# when running in github actions:
 source /home/runner/.rvm/scripts/rvm
+# when running locally in Act container:
+#source /usr/local/rvm/scripts/rvm
 
-rvm install ruby $ARGS
-rvm use $ARGS
+
+#rvm package install openssl # hack because ubuntu won't give us openSSL
+rvm install ruby $MATRIX_RUBY_VERSION
+rvm use $MATRIX_RUBY_VERSION
 gem install bundler
 
-(cd spec/rails5; bundle install; yard gems)
-(cd spec/rails6; bundle install; yard gems)
-(cd spec/rails7; bundle install; yard gems)
+(cd spec/rails${MATRIX_RAILS_VERSION}; bundle install; yard gems)
 
 bundle install
-bundle exec rspec
+
+bundle exec rspec spec/solargraph-rails/annotate_spec.rb  spec-solargraph-rails/delegate_spec.rb  spec/solargraph-rails/model_spec.rb      spec/solargraph-rails/schema_spec.rb spec/solargraph-rails/autoload_spec.rb  spec/solargraph-rails/devise_spec.rb    spec/solargraph-rails/rails5_spec.rb spec/solargraph-rails/storage_spec.rb
```

Comment:
hardcoding this until I get it working for at least ruby 2.7 / rails 5; then I'll make it find the tests properly